package.json

```diff
@@ -41,12 +41,12 @@
     "cpx": "1.5.0",
     "decompress-cli": "2.0.0",
     "enzyme": "2.7.0",
-    "eslint": "3.9.1",
+    "eslint": "4.8.0",
```

Comment:
Oh hey, `4.9.0` is available with my fix. https://eslint.org/blog/2017/10/eslint-v4.9.0-released